Gemfile

```diff
@@ -1,13 +0,0 @@
-source "https://rubygems.org"
```

Comment:
please do not remove those files

Reply:
Hi Bjorn, I used to have them in my gitignore. Sorry for the inconvenience I re-added them now